src/Bicep.Core/PrettyPrint/PrettyPrinter.cs

```diff
@@ -37,6 +37,18 @@ public static string PrintProgram(ProgramSyntax programSyntax, PrettyPrintOption
             return sb.ToString();
         }
 
+        public static string PrintSyntax(SyntaxBase syntax, PrettyPrintOptions options)
+        {
+            string indent = options.IndentKindOption == IndentKindOption.Space ?  new string(' ', options.IndentSize) : "\t";
+
+            var sb = new StringBuilder();
+            var documentBuildVisitor = new DocumentBuildVisitor();
+
+            var document = documentBuildVisitor.BuildDocument(syntax);
+            document.Layout(sb, indent, Environment.NewLine);
```

Comment:
>Environment.NewLine [](http://example.com/codeflow?start=40&length=19)

I wouldn't expect this method to ignore the newline setting in options. Can you add an Environment or Auto value to the enum in options to get the same behavior for both?

Reply:
This still hasn't been addressed.